cpp-linter/tests/paginated_changed_files.rs

```diff
@@ -38,14 +56,20 @@ async fn get_paginated_changes(lib_root: &Path, event_type: EventType) {
     let tmp = TempDir::new().expect("Failed to create a temp dir for test");
     let mut event_payload = NamedTempFile::new_in(tmp.path())
         .expect("Failed to spawn a tmp file for test event payload");
-    env::set_var("GITHUB_EVENT_PATH", event_payload.path());
-    if let EventType::PullRequest(pr_number) = event_type {
+    env::set_var(
+        "GITHUB_EVENT_PATH",
+        if test_params.no_event_payload {
+            Path::new("no a file.txt")
+        } else {
+            event_payload.path()
+        },
+    );
```

Comment:
I did. That's the purpose of `no_event_payload` parameter. "Silent" error logs are emitted from `rest_api/github/specific_api.rs` module, and an invalid sentinel value is used when this occurs. We cannot unset the env var `GITHUB_EVENT_PATH` because GitHub CI runners don't allow that.